types/Cargo.toml

```diff
@@ -51,9 +50,10 @@ getrandom = { version = "0.2.10", features = ["js"] }
 wasm-bindgen-test = "0.3"
 
 [features]
-default = ["p2p"]
+default = ["p2p", "std"]
 p2p = ["dep:libp2p-identity", "dep:multiaddr", "dep:serde_repr"]
 test-utils = ["dep:ed25519-consensus", "dep:rand"]
+std = []
```

Comment:
We need to disable `std` feature in dependencies too and enable them in our `std` feature. This list is incomplete list:

```
std = ["celestia-tendermint/std", "cid/std", "serde/std"]
```